### ISSUE TITLE
[Docs][Accessibility]: v4 docs OAuth diagram poor visibility

### ISSUE THREAD

**Issue by jwc-projects** [nextauthjs/next-auth#13295]
### What is the improvement or update you wish to see?

The v4 docs page on OAuth providers renders a diagram of an OAuth workflow with poor color contrast in dark mode. Update the colors of the affected text to meet minimum AA standards for accessible text.

### Is there any context that might help us understand?

The v4 docs page renders a SVG diagram of an OAuth workflow. The SVG text elements do not update their color to suit the users selected theme (dark mode/light mode). Instead these elements render text with color \#333 on a black background, resulting in a color contrast ratio of 1.66:1. The minimum ratio for color contrast for text is 4.5:1 with some exceptions (See WCAG Success Criterio 1.4.3). Observed 10/21/25 via Google Chrome Version 140.0.7339.133

### Does the docs page already exist? Please link to it.

https://next-auth.js.org/configuration/providers/oauth
